capabilities/runtime.md

```diff
@@ -0,0 +1,22 @@
+---
+displayName: Serverless Runtime
+epicsLabels:
+  - area/serverless
+---
+<!-- above metadata will be used on kyma.project.io page to display nice name of capability and have a reference to label that should be used while fetching from ZenHub/GitHub the information about related Epics and their delivery plan   -->
+
+## Scope
+
+The Serverless Runtime capability offering is a central part of the serverless strategy of Kyma. It is the easiest way to run custom code in Kyma and to integrate different services provided by the Services Broker and the Application Connector. Following the "batteries included" rule of Kyma, the function-as-a-service (FaaS) solution based on Knative is provided inside the OSS Project. Apart from having the FaaS solution, it is also possible to schedule workloads on third-party offerings using Knative.
```

Comment:
One thing I missed previously and wanted to clarify: did you mean Services Broker or Service Broker/Service Brokers actually? Please update accordingly.